modules/sf_h1nobbdde.py

```diff
@@ -44,13 +44,11 @@ def setup(self, sfc, userOpts=dict()):
             self.opts[opt] = userOpts[opt]
 	
     def watchedEvents(self):
-        return ["INTERNET_NAME"]
+        return ["DOMAIN_NAME"]
 
     # What events this module produces
     def producedEvents(self):
-        ret = ["VULNERABILITY"]
-
-        return ret
+         return ["*"]
```

Comment:
The module needs to return `VULNERABILITY`, you had it right there the first time.